_episodes/08-vector-open-shapefile.md

```diff
@@ -75,101 +70,121 @@ Key metadata for all shapefiles include:
+Datum: Amersfoort
+- Ellipsoid: Bessel 1841
 - Prime Meridian: Greenwich
 ~~~
 {: .output}
 
-Our data is in the CRS **UTM zone 18N**. The CRS is critical to 
+Our data is in the CRS **RD New**. The CRS is critical to 
 interpreting the object's extent values as it specifies units. To find
-the extent of our AOI in the projected coordinates, we can use the `.bounds()` function: 
+the extent of our dataset in the projected coordinates, we can use the `.total_bounds` function: 
 
 ~~~
-aoi_boundary_HARV.bounds
+cropfield.total_bounds
 ~~~
 {: .language-python}
 
 ~~~
-            minx          miny           maxx          maxy
-0  732128.016925  4.713209e+06  732251.102892  4.713359e+06
+array([ 13653.6128, 306851.867 , 277555.288 , 612620.9868])
 ~~~
 {: .output}
 
-The spatial extent of a shapefile or `shapely` spatial object represents the geographic "edge" or location that is the furthest north, south, east, and west. Thus, it is represents the overall geographic coverage of the spatial object. Image Source: National Ecological Observatory Network (NEON).
+This array contains, in order, the values for minx, miny, maxx and maxy, for the overall dataset. The spatial extent of a shapefile or `shapely` spatial object represents the geographic "edge" or location that is the furthest north, south, east, and west. Thus, it is represents the overall geographic coverage of the spatial object. Image Source: National Ecological Observatory Network (NEON).
 
 ![Extent image](../fig/dc-spatial-vector/spatial_extent.png)
 
 We can convert these coordinates to a bounding box or acquire the index of the dataframe to access the geometry. Either of these polygons can be used to clip rasters (more on that later). 
 
-## Reading a Shapefile from a csv
+## Cropping a Shapefile
+Our `cropfield` dataset is rather large, containing data for the entirety of the European portion of the Netherlands. Before plotting it we will first select a specific section of to be our area of interest.
+
+We can create a cropped version of our dataset as follows:
+```python
+# Define a Boundingbox in RD
+xmin, xmax = (120000, 135000)
+ymin, ymax = (485000, 500000)
+cropfield_crop = cropfield.cx[xmin:xmax, ymin:ymax]
+```
 
-So far we have been loading file formats that were specifically built to hold spatial information. But often, point data is stored in table format, with a column for the x coordinates and a column for the y coordinates. The easiest way to get this type of data into a GeoDataFrame is with the `geopandas` function `geopandas.points_from_xy`, which takes list-like sequences of x and y coordinates. In this case, we can get these list-like sequences from columns of a pandas `DataFrame` that we get from `read_csv`.
+This will cut out a smaller area, defined by a box in units of the projection, discarding the rest of the data. The resultant GeoDataframe is found in the `cropfield_crop` object. We can check the total bounds of this new data as before:
```

Comment:
I am wondering whether we should add something like the following to make clear that the `.cx` indexer selects all the features intersecting the provided spatial range? It might otherwise look counterintuitive that the `total_bounds` of the cropped object does not match the specified bbox exactly - what do you think?
```suggestion
This will cut out a smaller area, defined by a box in units of the projection, discarding the rest of the data. The resultant GeoDataframe, which includes all the features intersecting the box, is found in the `cropfield_crop` object. We can check the total bounds of this new data as before:
```

Reply:
No, you are completely right - it's not "cropping" in the full sense since edge elements are themselves not cropped. It's more a type of selection. I have made your suggested change, and added an extra sentence ("Note that the edge elements are not 'cropped' themselves.") to hopefully clarify this a bit further. Do you think we need to make it clearer...?

Reply:
Great, I think that now it is super clear!